onnxruntime/core/session/provider_bridge_ort.cc

```diff
@@ -1466,6 +1466,14 @@ ProviderOptions GetProviderInfo_Cuda(const OrtCUDAProviderOptionsV2* provider_op
   return s_library_cuda.Get().GetProviderOptions(reinterpret_cast<const void*>(provider_options));
 }
 
+void UpdateProviderInfo_Rocm(OrtROCMProviderOptions* provider_options, const ProviderOptions& options) {
+  return s_library_rocm.Get().UpdateProviderOptions(reinterpret_cast<void*>(provider_options), options);
+}
+
```

Comment:
Is this being called from the provider side?

Reply:
Redundant function removed.